src/launcher_gui.js

```diff
@@ -21,89 +26,123 @@ app.on('second-instance', () => {
 });
 
 app.prependListener('ready', () => {
-	const { wizard } = require('./launcher_wizard');
-
-	const display = electron.screen.getPrimaryDisplay();
-	const sWidth = display.workAreaSize.width;
-	const width = 800;
-	const height = process.platform === 'win32' ? 418 : 380 + 8;
-
-	let windowOpts = {
-		x: (sWidth - width) / 2,
-		// y: (sHeight - height) / 2,
-		y: 100,
-		width: width,
-		height: height,
-		show: false,
-		icon: `${__dirname}/renderer/spring-icon.png`,
-		webPreferences: {
-			nodeIntegration: true,
-			enableRemoteModule: true,
-		},
-	};
-	windowOpts.resizable = false;
-	Menu.setApplicationMenu(null);
-	mainWindow = new BrowserWindow(windowOpts);
-
-	mainWindow.loadFile(`${__dirname}/renderer/index.html`);
-	if (isDev) {
-		// mainWindow.webContents.openDevTools();
-	}
+	// TODO: is there a better way to get yargs' positional args ? Did not find
+	// it ...
+	if (argv._[0]) {
```

Comment:
it's indeed quite ugly and it might be a good idea to overwrite the argv object (simply set `argv.replayFile = argv._[0]`), in `launcher_args.js`

it's better to confine such positional argument-parsing logic in one place, and it might give us some more flexibility if we ever add other positional arguments (e.g. URI parsing to automatically connect to a spring game or launch spring)

Reply:
I'll do that, it will be much better indeed. I'm still surprised that a mature command line parsing library like yargs doesn't have a better way to do that ...